exec/totemknet.c

```diff
@@ -57,9 +57,13 @@
 #include <sys/poll.h>
 #include <sys/uio.h>
 #include <limits.h>
+#include <libgen.h>
```

Comment:
minor nitpick here, but all those includes that we are adding to support nozzle, shouldn´t they be within #ifdef HAVE_LIBNOZZLE ?